inst/json/impact_docs.html

```diff
@@ -0,0 +1,48 @@
+<p><strong>Impact</strong> shows the estimated prevalence in the zone over time and the average clinical cases averted
```

Comment:
This is great for now, but wondering if in terms of maintainability by researchers if we moved this to md and compile to html (non-standalone) at startup?

Reply:
Great idea